examples/php.py

```diff
@@ -0,0 +1,9 @@
+import cpmpy as cp
```

Comment:
should have at least a little comment on it being the pigenhole problem...

should not be part of solver commit, OK to add to examples